sktime/datatypes/_vectorize.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Wrapper for easy vectorization/iteration of time series data.
+
+Contains VectorizedDF class.
+"""
+
+import pandas as pd
+
+from sktime.datatypes._check import check_is_scitype, mtype
+from sktime.datatypes._convert import convert_to
+
+
+class VectorizedDF:
+    """Wrapper for easy vectorization/iteration over instances.
+
+    VectorizedDF is an iterable that returns pandas.DataFrame
+        in sktime Series or Panel format.
+    Elements are all Series or Panels in X, these are iterated over.
+
+    Parameters
+    ----------
+    X : Panel or Hierarchical data container, to vectorize over
```

Comment:
Type and description need to seperated by - or new line

Reply:
fixed